File: Data Structures & Algorithms/daily-temperatures/submission-0.java
class Solution {
    public int[] dailyTemperatures(int[] temperatures) {
        Stack<int[]> stack=new Stack<>();
        int[] ans=new int[temperatures.length];
        for(int i=0;i<temperatures.length;i++){
            int t=temperatures[i];
            while(!stack.isEmpty() && t>stack.peek()[0]){
                int[] temp=stack.pop();
                ans[temp[1]]=i-temp[1];
            }
            stack.push(new int[]{t,i});
        }
        return ans;
    }
}

File: Data Structures & Algorithms/subsets/submission-0.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        ans=[]
        def rec(i,temp):
            if i>len(nums)-1:
                ans.append(temp[:])
                return 
            temp.append(nums[i])
            rec(i+1,temp)
            temp.pop()
            rec(i+1,temp)
        rec(0,[])
        return ans